template/.github/workflows/deploy.yml

```diff
@@ -0,0 +1,33 @@
+name: Netlify Release
+on:
+  push:
+    branches: ['main', 'master']
+  workflow_dispatch:
+    inputs:
+      deploy-msg:
+        description: 'Deploy Message (visible in Netlify)'
+        required: true
+
+jobs:
+  deploy:
```

Comment:
I think we're missing the name of the jobs 🤔

Reply:
50c2b66 ✅